src/Symfony/Component/Form/Extension/Core/Type/NumberType.php

```diff
@@ -26,10 +28,21 @@ public function buildForm(FormBuilderInterface $builder, array $options)
         $builder->addViewTransformer(new NumberToLocalizedStringTransformer(
             $options['scale'],
             $options['grouping'],
-            $options['rounding_mode']
+            $options['rounding_mode'],
+            $options['html5'] ? 'en' : null
```

Comment:
shouldnt we expose the locale option instead? Makes sense for number formatting being locale aware.

Then we can be future proof, and render type=number for any supporting locale (`en === locale` of check from Intl maybe)

Reply:
I am not sure. I mean I would not expose it here to solve this particular use case. Being able to configure the locale to be used should then IMO be made possible for all form types that use locale depending formatters. But this then only makes sense if you need to change the locale per type. Not sure that there are any use cases for that. To set it globally using the `Locale` class IMO sounds better then introducing yet another config option.

Reply:
> Not sure that there are any use cases for that.

indeed, i guess not :D in practice you'll always input normalized numbers (in a single locale).